helm/ovn-kubernetes/values.yaml

```diff
@@ -76,6 +76,8 @@ global:
   enableMultiExternalGateway: true
   # -- Configure to use stateless network policy feature with ovn-kubernetes
   enableStatelessNetworkPolicy: false
+  # -- Configure to use the IPAMClaims CRD feature with ovn-kubernetes, thus granting persistent IPs across restarts / migration for KubeVirt VMs
+  enablePersistentIPs: false
```

Comment:
hmmm... @maiqueb for helm, I'm used to seeing the values deed to this file influence something in the template somewhere in the chart. I am not seeing it here, other then this value by itself. Was that intentional?

As a simple and random example, take a look of how `enableStatelessNetworkPolicy` value is used:

```
$ grep -C2 -r 'enableStatelessNetworkPolicy' --include "*"
ovn-kubernetes/README.md-               </tr>
ovn-kubernetes/README.md-               <tr>
ovn-kubernetes/README.md:                       <td>global.enableStatelessNetworkPolicy</td>
ovn-kubernetes/README.md-                       <td>bool</td>
ovn-kubernetes/README.md-                       <td><pre lang="json">
--
ovn-kubernetes/charts/ovnkube-master/templates/deployment-ovnkube-master.yaml-          value: {{ default 20 .Values.global.aclLoggingRateLimit | quote }}
ovn-kubernetes/charts/ovnkube-master/templates/deployment-ovnkube-master.yaml-        - name: OVN_STATELESS_NETPOL_ENABLE
ovn-kubernetes/charts/ovnkube-master/templates/deployment-ovnkube-master.yaml:          value: {{ hasKey .Values.global "enableStatelessNetworkPolicy" | ternary .Values.global.enableStatelessNetworkPolicy false | quote }}
ovn-kubernetes/charts/ovnkube-master/templates/deployment-ovnkube-master.yaml-        - name: OVN_ENABLE_MULTI_EXTERNAL_GATEWAY
ovn-kubernetes/charts/ovnkube-master/templates/deployment-ovnkube-master.yaml-          value: {{ hasKey .Values.global "enableMultiExternalGateway" | ternary .Values.global.enableMultiExternalGateway false | quote }}
--
ovn-kubernetes/values.yaml-  enableMultiExternalGateway: true
ovn-kubernetes/values.yaml-  # -- Configure to use stateless network policy feature with ovn-kubernetes
ovn-kubernetes/values.yaml:  enableStatelessNetworkPolicy: false
ovn-kubernetes/values.yaml-  # -- Configure to use the IPAMClaims CRD feature with ovn-kubernetes, thus granting persistent IPs across restarts / migration for KubeVirt VMs
ovn-kubernetes/values.yaml-  enablePersistentIPs: false
```